Fabric.Terminology.API/Scripts/Terminology-Install-Utilities.psm1

```diff
@@ -130,10 +130,11 @@ function Invoke-SqlCommand {
         [String] $SqlServerAddress,
         [String] $DatabaseName,
         [String] $Query,
-        [PSCustomObject] $Parameters = @{}
+        [PSCustomObject] $Parameters = @{},
+        [Boolean] $ReturnData = $false
```

Comment:
You could make this a [switch] and it would be $false by default. Not a big deal, but it would be a little cleaner

Reply:
I think I would prefer to leave it how it is.  It is a little more explicit and easier (for me at least) to understand.